elasticapm/__init__.py

```diff
@@ -60,5 +60,7 @@
     VERSION = "unknown"
 
 
-if sys.version_info >= (3, 5):
-    from elasticapm.contrib.asyncio.traces import async_capture_span  # noqa: F401
+if sys.version_info <= (3, 5):
+    raise DeprecationWarning("The agent requires python 3.6+")
```

Comment:
Doesn't hurt to be a bit more verbose here
```suggestion
    raise DeprecationWarning("The Elastic APM agent requires Python 3.6+")
```